extra/exe-builder/Program.cs

```diff
@@ -98,6 +109,12 @@ public class UptimeKumaApplicationContext : ApplicationContext
         }
 
         void DownloadFiles() {
+            if (intranetOnly)
+            {
+                MessageBox.Show("You have parsed in --intranet so we will not download any files from the internet. If you run into issues remember that the initial run needs to be without --intranet.");
```

Comment:
`The --intranet argument was provided, so we will not try to access the internet.` should already be in the log => I don't see why logging this again has additional value

```suggestion
```